silq/pulses/pulse_sequences.py

```diff
@@ -1217,36 +1211,24 @@ class NMRCPMGPulseSequence(NMRPulseSequence):
               successively apply. Can be strings, in which case the string
               should be an item in ``NMR`` whose value is a `Pulse`. Default is
               single element ``NMRPulseSequence.NMR['NMR_pulse']``.
+
+              NOTE THAT THE FIRST PULSE IN THIS STRING NEEDS TO BE  A PI/2 NMR PULSE,i.e
+              NMR['NMR_pulses'] = [['NMR_pi_half']+N*['NMR_pi']+['NMR_pi_half']]
+
```

Comment:
I don't think this comment is necessary.

Reply:
I guess after writing the default pulse sequence in the NMR pulse settings we can get rid of this comment